$OpenBSD: patch-OpenSSL_test_test_ssl_py,v 1.2 2015/07/20 07:02:58 sthen Exp $
--- OpenSSL/test/test_ssl.py.orig	Sun Feb 23 05:36:45 2014
+++ OpenSSL/test/test_ssl.py	Fri Jul 17 22:17:01 2015
@@ -1576,13 +1576,13 @@ class ConnectionTests(TestCase, _LoopbackMixin):
             errno instead of raising an exception.
             """
             port = socket()
-            port.bind(('', 0))
+            port.bind(('127.0.0.1', 0))
             port.listen(3)
 
             clientSSL = Connection(Context(TLSv1_METHOD), socket())
             clientSSL.setblocking(False)
             result = clientSSL.connect_ex(port.getsockname())
-            expected = (EINPROGRESS, EWOULDBLOCK)
+            expected = (0, EINPROGRESS, EWOULDBLOCK)
             self.assertTrue(
                     result in expected, "%r not in %r" % (result, expected))
 
